File: practica1/prac1.py
import math

class Punto:
    def __init__(self, x, y):
        self.x = float(x)
        self.y = float(y)

    def coord_cartesianas(self):
        return (self.x, self.y)

    def coord_polares(self):
        r = math.sqrt(self.x**2 + self.y**2)
        theta = math.atan2(self.y, self.x)
        return (r, theta)

    def toString(self):
        return "Punto(" + str(self.x) + ", " + str(self.y) + ")"

# Prueba de escritorio
p = Punto(3, 4)
print(p.toString())  
print("Coordenadas Cartesianas:", p.coord_cartesianas())  
print("Coordenadas Polares:", p.coord_polares())  

File: practica1/prac1.java
public class Punto {
	 private float x;
	    private float y;

	    public Punto(float x, float y) {
	        this.x = x;
	        this.y = y;
	    }

	    public float[] coordCartesianas() {
	        return new float[]{x, y};
	    }

	    public double[] coordPolares() {
	        double r = Math.sqrt(x * x + y * y);
	        double theta = Math.atan2(y, x);
	        return new double[]{r, theta};
	    }

	    public String toString() {
	        return "Punto(" + x + ", " + y + ")";
	    }

	    public static void main(String[] args) {
	        Punto p = new Punto(3, 4);
	        System.out.println(p.toString());
	        float[] cartesianas = p.coordCartesianas();
	        System.out.println("Coordenadas Cartesianas: (" + cartesianas[0] + ", " + cartesianas[1] + ")");
	        double[] polares = p.coordPolares();
	        System.out.println("Coordenadas Polares: (" + polares[0] + ", " + polares[1] + ")");
	    }